code/game/objects/machinery/vending/new_marine_vendors.dm

```diff
@@ -87,7 +90,7 @@
 		to_chat(H, "<span class='warning'>Wrong ID card owner detected.</span>")
 		return
 
-	if(vendor_role && I.rank != vendor_role)
+	if(lock_flags & JOB_LOCK && vendor_role && I.rank != vendor_role)
```

Comment:
Can't remember if we wanted to use the macros for bitchecks or not. maybe someone else knows.

Reply:
It's 100% optional to use these or not.

Reply:
After ninjanomnom reeed enough, I've begun skipping the macros in places that are not new-coder territory.